index.d.ts

```diff
@@ -3303,6 +3304,11 @@ declare module 'mongoose' {
       type?: string): this;
   }
 
+  export interface SyncIndexesOptions {
```

Comment:
`SyncIndexesOptions` needs to at minimum support the same options as `createIndexes()`. Maybe try `type SyncIndexesOptions = mongodb.CreateIndexesOptions & { continueOnError?: boolean }`?